test/unit/app/controllers/metametrics-test.js

```diff
@@ -0,0 +1,546 @@
+import { strict as assert } from 'assert'
+import sinon from 'sinon'
+import MetaMetricsController from '../../../../app/scripts/controllers/metametrics'
+import { ENVIRONMENT_TYPE_BACKGROUND } from '../../../../app/scripts/lib/enums'
+import { createSegmentMock } from '../../../../app/scripts/lib/segment'
+import {
+  METAMETRICS_ANONYMOUS_ID,
+  METAMETRICS_BACKGROUND_PAGE_OBJECT,
+} from '../../../../shared/constants/metametrics'
+import waitUntilCalled from '../../../lib/wait-until-called'
+
+const segment = createSegmentMock(2, 10000)
```

Comment:
It looks like these two are being used as shared mocks. It would be better to construct these per-test, so that there can be no chance of tests affecting each other.

Reply:
Oh I see, it's being flushed after each test :thinking: